### PR TITLE
Finalising Profile Navigation 

### PR DESCRIPTION
**Pull request by ktm07**
### **Profile.tsx**
It had layout of the profile page and "DummyArray" where all photos are stored for gallery.

### **ProfileNavigator.tsx**
I used stack navigator to store screens, to move from edit page to profile page and reverse.

###  ProfileGallery.tsx
It is a layout of Gallery component which used in profile page, it is scrollable. 

Screenshot 2021-01-22 at 2 26 43 pm

Screenshot 2021-01-22 at 2 26 31 pm

Screenshot 2021-01-22 at 2 53 27 pm

### CODE REVIEW

File: src/Components/Profile/EditProfile.tsx [hunk 1]
```diff
@@ -0,0 +1,293 @@
+import { Alert, ScrollView, StyleSheet, Text, TextInput, View, } from "react-native";
+import React, { useState } from "react";
+import { Button, Icon } from "native-base";
+import ImagePickerr from "../create-post/ImagePicker";
+import { UserObject } from "../../Types/types";
+import mockUser from "../../mock/mockUser";
+
+interface EditProfileProps {
+    profile: string;
```
Comment (wrt95) [inline src/Components/Profile/EditProfile.tsx:9]: Can you delete this line 9? 
It is not needed.

File: src/Components/create-post/CreatePost.tsx [hunk 2]
```diff
@@ -65,14 +65,14 @@ const CreatePost = (props: CreatePostProps) => {
...
-        : Alert.alert("Sorry, something wrong happened")
+            // If there is no image, alert the user.
+            : !imageSelected ?
+                Alert.alert("You must upload an image before pressing create")
+                // If length of the title is less than 2, alert the user.
+                : title.length < 2 ?
+                    Alert.alert("Sorry, the length of your title must be 2 or more characters.")
+                    // Else give error message to the user
+                    : Alert.alert("Sorry, something wrong happened")
     };
```
Comment (wrt95) [inline src/Components/create-post/CreatePost.tsx:76]: Can you edit the tabs in this file? So it is on the correct places? Look at how it was before.

**Review by wrt95 — CHANGES_REQUESTED** [review]

File: src/Components/Profile/EditProfile.tsx (hunk 1 above)
Comment (ktm07) [reply src/Components/Profile/EditProfile.tsx]: Yeah I removed that line

File: src/Components/create-post/CreatePost.tsx (hunk 2 above)
Comment (ktm07) [reply src/Components/create-post/CreatePost.tsx:76]: Tab spaces are automatically taken in vs code, so I think it's not a big issue.

Comment (wrt95) [reply src/Components/create-post/CreatePost.tsx:76]: Okey, thats fine ;)

**Review by wrt95 — APPROVED** [review]

**Review by sp103 — APPROVED** [review]
All looks good to me

**Pull request merged**